setup.py

```diff
@@ -1,4 +1,14 @@
 from setuptools import setup, find_packages
+import os
+
+cmmnd1 = "python domain_setup.py build_ext " \
+         "--build-lib=jyulb/internal/common/"
+
+cmmnd2 = "python solver_setup.py build_ext " \
+         "--build-lib=jyulb/internal/isothermal/"
+
+os.system(cmmnd1)
```

Comment:
It is not a good idea to do these operations while a setup.py is processed. I would suggest to remove this code and include here the extension declarations as described in https://github.com/simphony/simphony-jyulb/pull/15#issuecomment-92342335